Sources/TuistCore/Graph/ModelExtensions/Target+Core.swift

```diff
@@ -78,7 +78,7 @@ extension Target {
             Set(paths)
                 .subtracting(excluded)
                 .filter { path in
-                    if let `extension` = path.extension, Target.validSourceExtensions.contains(`extension`) {
+                    if let `extension` = path.extension, Target.validSourceExtensions.contains(where: { $0.caseInsensitiveCompare(`extension`) == .orderedSame }) {
                         return true
                     }
                     return false
```

Comment:
Can't this be just:
```
return Target.validSourceExtensions.contains(where: { $0.caseInsensitiveCompare(path.extension) == .orderedSame }) 
```
?

Reply:
You can do `path.extension?.case insensitive...`
Btw no strong opinion, you can even keep it as it is 👌